src/testBoost.cpp

```diff
@@ -0,0 +1,26 @@
+#define BOOST_TEST_DYN_LINK
```

Comment:
Create subfolders with proper test projects for all gtest/boost/catch frameworks, with a main file consisting the main() function and at least two other files that contain tests (both including ecfeed.hpp). You can build them with conan/cmake.

Reply:
Done + Conan/CMake. In Google, according to the documentation, 'main' is considered as boilerplate and, providing that some massive changes in functionality are not needed, it should be omitted. Here, it is commented (but works). Catch2 works well with main. I'll have to do some more research with Boost. Here, 'main' is still boilerplate but I don't know how to write a custom one (examples omit it, and when added the 'redefinition of ‘int main(int, char**)' error appears).

Reply:
Ok, it seems not that importand in these exmples. Normally I add main function in google test, to do some additional setup, parse command line arguments and so on. Apparently not needed here. Enough to see that many source files work.